src/uk/gov/defra/ffc/Build.groovy

```diff
@@ -13,6 +13,7 @@ class Build implements Serializable {
   private static def verifyCommitBuildable(ctx, pr, defaultBranch) {
     if (pr) {
       ctx.echo("Building PR$pr")
+      ctx.env.PR_BUILD = true
```

Comment:
Enables an equivalent env var to be set for use in a container e.g. acceptance tests.